.github/workflows/test-mongodb.yml

```diff
@@ -0,0 +1,32 @@
+---
+name: test with mongodb
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        entry:
+          - { ruby: 2.6.2, mongoid: 6.4.8, mongodb: 4.4 }
+          - { ruby: 2.6.2, mongoid: 7.2.3, mongodb: 4.4 }
+          - { ruby: 2.6.2, mongoid: 7.3.0, mongodb: 4.4 }
+    name: test (ruby=${{ matrix.entry.ruby }}, mongoid=${{ matrix.entry.mongoid }}, mongodb=${{ matrix.entry.mongodb }})
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.entry.ruby }}
+      - uses: browser-actions/setup-geckodriver@latest
+      - run: geckodriver --version
+      - uses: supercharge/mongodb-github-action@1.7.0
+        with:
+          mongodb-version: ${{ matrix.entry.mongodb }}
+      - name: Test
+        uses: GabrielBB/xvfb-action@v1
```

Comment:
xvfb is for tests that use a browser, remove and replace with a plain `run`

Reply:
done